src/Compilers/Core/Portable/DiagnosticAnalyzer/SuppressMessageAttributeState.cs

```diff
@@ -75,21 +74,6 @@ public SuppressMessageAttributeState(Compilation compilation)
 
         public bool IsDiagnosticSuppressed(Diagnostic diagnostic, ISymbol symbolOpt = null)
```

Comment:
ignore the comment, I didn't see it has "return IsDiagnosticSuppressed(diagnostic.Id, diagnostic.Location); " this as well.